cnd/src/network.rs

```diff
@@ -1131,6 +1137,185 @@ impl libp2p::swarm::NetworkBehaviourEventProcess<()> for ComitNode {
     fn inject_event(&mut self, _event: ()) {}
 }
 
+impl libp2p::swarm::NetworkBehaviourEventProcess<comit_ln::BehaviourOutEvent> for ComitNode {
+    fn inject_event(&mut self, event: comit_ln::BehaviourOutEvent) {
+        match event {
+            comit_ln::BehaviourOutEvent::SwapIsFinalizedCanStartExecution {
+                local_swap_id,
+                swap_params,
+                secret_hash,
+                invoice_states,
+                ethereum_identity,
+                ethereum_ledger_state,
+            } => {
+                let role = swap_params.role;
+                let create_swap_params = swap_params;
+
+                if role == Role::Alice {
+                    tokio::task::spawn({
+                        let lnd_connector = (*self.lnd_connector_as_receiver)
+                            .clone()
+                            .read_certificate()
+                            .expect("Failure reading tls certificate")
+                            .read_macaroon()
+                            .expect("Failure reading macaroon");
+
+                        new_halight_swap(local_swap_id, secret_hash, invoice_states, lnd_connector)
+                            .instrument(
+                                tracing::error_span!("beta_ledger", swap_id = %local_swap_id, role = %role),
+                            )
+                    });
+                } else {
```

Comment:
This is not related to your code but I already noticed it in the past.
Could you replace the `if..else` with a `match` block on `role` as to avoid the "// This is Bob` comment?